From a6f4c186888387da739e0980eba4665c14d396a6 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 18 Apr 2024 13:54:49 +0200
Subject: [PATCH 0806/1695] drivers: stm32_cpu_opp: add missing inclusion in
 header file

Add missing inclusion of tee_api_types.h in stm32_cpu_opp.h

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4470d83e65cbc2be0ab10a2840ce9b321d41cdf3 drivers: stm32_cpu_opp: add missing inclusion in header file]
Change-Id: Iac655c9f0917e46e3ecfe45915566311280e56af
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/374682
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/include/drivers/stm32_cpu_opp.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/include/drivers/stm32_cpu_opp.h b/core/include/drivers/stm32_cpu_opp.h
index 1736aa5ec..f50a30a92 100644
--- a/core/include/drivers/stm32_cpu_opp.h
+++ b/core/include/drivers/stm32_cpu_opp.h
@@ -6,6 +6,8 @@
 #ifndef DRIVERS_STM32_CPU_OPP_H
 #define DRIVERS_STM32_CPU_OPP_H
 
+#include <tee_api_types.h>
+
 struct regulator;
 struct clk;
 
-- 
2.47.2

